.storybook/preview.js

```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters

> Setting a global parameter is a common way to configure addons.

So this config is consumed by the Storybook addons, is that right?

Reply:
Yes; specifically the [actions addon](https://storybook.js.org/docs/react/essentials/actions)